src/recipe/recipe.service.ts

```diff
@@ -18,6 +18,17 @@ export class RecipeService {
     return this.recipeModel.findById(id).lean().exec();
   }
 
+  getAll(paginatedQueryDto?: Partial<PaginatedQueryDto>): Promise<RecipeDto[]> {
+    const page = paginatedQueryDto?.page || 1;
+    const limit = paginatedQueryDto?.limit || 10;
```

Comment:
you should set default pagination options like `page` or `limit` in config file (for example `pagination_config.ts`) then in validation or another endpoint you can use the same default options.

Reply:
did you have validation for pagination options? 

if not please add it

Reply:
and add unit tests for it.